robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -21,7 +23,7 @@
   private Context context;
 
   public RoboMenu() {
-    this(null);
+    this(RuntimeEnvironment.application);
```

Comment:
Removing that constructor means that subclasses _must_ call super(Context) so I would argue that it is less explicit this way as the caller has no idea this class magically provides the application context for retrieving strings. As I said, different contexts can have different resources so its better that the user is required to pass it in. If you want to fail fast you can always add a Preconditions.checkNotNull() in the constructor.